GitCommands/Settings/AppSettings.cs

```diff
@@ -1651,7 +1651,10 @@ public static string GetGitExtensionsFullPath()
         [CanBeNull]
         public static string GetGitExtensionsDirectory()
         {
-            return Path.GetDirectoryName(GetGitExtensionsFullPath());
+            var assembly = Assembly.GetEntryAssembly() ?? // common case, GitExtensions.exe
```

Comment:
Should this be separated to a testaccessor?

Reply:
I don't see how that would help.
The problem I am solving here, is GetGitExtensionsDirectory() pointing to `TestRunner.exe` directory, while the application .dlls and other files remain in /bin/Debug as usual.